plugin/trino-druid/src/main/java/io/trino/plugin/druid/DruidJdbcClient.java

```diff
@@ -86,31 +85,25 @@ public DruidJdbcClient(BaseJdbcConfig config, ConnectionFactory connectionFactor
     @Override
     public Optional<JdbcTableHandle> getTableHandle(ConnectorSession session, SchemaTableName schemaTableName)
     {
-        try (Connection connection = connectionFactory.openConnection(session)) {
-            String jdbcSchemaName = schemaTableName.getSchemaName();
-            String jdbcTableName = schemaTableName.getTableName();
-            try (ResultSet resultSet = getTables(connection, Optional.of(jdbcSchemaName), Optional.of(jdbcTableName))) {
-                List<JdbcTableHandle> tableHandles = new ArrayList<>();
-                while (resultSet.next()) {
+        String jdbcSchemaName = schemaTableName.getSchemaName();
+        String jdbcTableName = schemaTableName.getTableName();
+        try (
+                Connection connection = connectionFactory.openConnection(session);
+                ResultSet resultSet = getTables(connection, Optional.of(jdbcSchemaName), Optional.of(jdbcTableName))
+        ) {
```

Comment:
nitpick but for consistency:

```suggestion
        try (Connection connection = connectionFactory.openConnection(session);
                ResultSet resultSet = getTables(connection, Optional.of(jdbcSchemaName), Optional.of(jdbcTableName))) {
```